src/System.Runtime.Handles/tests/System.Runtime.Handles.Tests.csproj

```diff
@@ -20,6 +20,7 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
```

Comment:
> Do we know why we have to explicitly specify these targets in the reference?

Because, by default, only the .dll is copied, and we need the .pdb copied as well, for both debugging and code coverage purposes.